backend/src/models/reports/genericModelTemplate.js

```diff
@@ -13,6 +13,10 @@
 
 // libs
 const { knex } = require("@database/databaseConnection")();
+// const _ = require("lodash"); // for data-type-agnostic list processing
+
+// utils
+// const { groupByProperty } = require("@controllers/helpers");
```

Comment:
More commented-out code

Reply:
this template isn't connected to any working code. the commented out imports are here as a courtesy to the next developer, but I can understand why you'd find it rendundant